### DISCUSSION TITLE
Add plugins in Security onion 2.3.6

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#5622] [Unsupported Versions] [answered]
Isit possible or way to add plugins in Security onion 2.3.6 like https://github.com/Security-Onion-Solutions/security-onion/wiki/Kibana#plugins ? I want to add wazuh ...Kindly guide me if possible

**Comment by lan94** [comment, marked as answer]
It is possible to install and use the wazuh kibana plugin within Security Onion. However, the problem is that the latest version of the wazuh kibana plugin also requires the new version of wazuh (4.2.2), while Security Onion still uses an old wazuh version (3.13). So updating wazuh by yourself sometimes leads to chaos, because wazuh gets downgraded again with every update (which can break the wazuh configuration), so you always have to fix wazuh and update it again.
The best way I know so far is to wait until[ the team is able to update](https://github.com/Security-Onion-Solutions/securityonion/issues/1748) wazuh to the latest version.

**Reply by usmanaa10** [reply]
ok can you give me a solution , I am looking for a dashboard of wazuh which give statsof each module of wazuh 
e.g shown in image
![image](https://user-images.githubusercontent.com/66419518/134324463-cceae270-6a84-4a19-9807-bfe5eb70e32a.png)

**Reply by lan94** [reply]
You could try configuring wazuh manager to use the[ same index template](https://raw.githubusercontent.com/wazuh/wazuh/v4.2.1/extensions/elasticsearch/7.x/wazuh-template.json) that the Kibana plugin uses. With the fields in this template, you should be able to create dashboards that closely resemble the dashboards in the Kibana plugin.

**Reply by usmanaa10** [reply]
Thanks , can share steps how should i do , beacuse i dont want to make mistake at this stage...Please
